hcontext/requestid.go

```diff
@@ -15,30 +15,34 @@ import (
 
 type idkey int
 
-var key idkey
+var ridKey idkey
 
 var headersToSearch = []string{
-	"Request-ID", "X-Request-ID",
+	"Request-Id", "X-Request-Id",
```

Comment:
Is this a breaking change?

Reply:
Oh god, everything is awful. I'm just gonna specify both.